freqtrade/rpc/api_server/ws/channel.py

```diff
@@ -34,6 +35,8 @@ def __init__(
         self._serializer_cls = serializer_cls
 
         self._subscriptions: List[str] = []
+        self.queue: asyncio.Queue[Dict[str, Any]] = asyncio.Queue(maxsize=32)
```

Comment:
this means we have at max. 32 messages per consumer in the queue, right?

Reply:
Yes, the internal queue the `websockets` package uses defaults to 32 as well, so this is set to match that.